examples/desktop/image_widget/image_widget_single_video.py

```diff
@@ -0,0 +1,47 @@
+"""
+Image widget Video
+==================
+
+Example showing how to scroll through one or more videos using the ImageWidget
+"""
+
+# test_example = true
+# sphinx_gallery_pygfx_docs = 'animate 6s 20fps'
+
+import fastplotlib as fpl
+import imageio.v3 as iio
+import numpy as np
+
+
+movie = iio.imread("imageio:cockatoo.mp4")
+
+# Ignore and do not use the next 2 lines
+# for the purposes of docs gallery generation we subsample and only use 15 frames
+movie_sub = movie[:15, ::12, ::12].copy()
+del movie
```

Comment:
@clewis7 until we either figure out a way to hide this from the docs example code (maybe do it under the hood in ImageWidget by checking for `RTD_BUILD=1` env variable) or fix the annoying sphinx gallery memory leak

Reply:
I think we do it under the hood by checking for env variable